docusaurus/docs/deployment.md

```diff
@@ -312,7 +313,7 @@ Then run:
 npm run deploy
 ```
 
-### Step 4: Ensure your project’s settings use `gh-pages`
+### Step 4: For a project's page, ensure your project’s settings use `gh-pages`

```

Comment:
Should this say `project page`? That's how it looks above.

Reply:
I added `project page` from the understanding that normal users might otherwise think that `Step 4` applies for both Personal and Project's page.

Let me know if you'd want to keep `project page` and I'd proceed to merge/rebase. 

Thanks.

Reply:
I think we can change `projects's` to `project`

Reply:
Have made the change and passed both tests. 

Thanks.